ovirt_imageio/_internal/auth.py

```diff
@@ -145,10 +151,10 @@ def add_context(self, con_id, context):
         with self._lock:
             if self._canceled:
                 raise errors.AuthorizationError(
-                    "Ticket {} was canceled".format(self.uuid))
+                    "Transfer {} was canceled".format(self.transfer_id))
```

Comment:
Sorry, a little bit confused, shouldn't it be "self._transfer_id" (with a preceding underscore) as in lines 57-58?

Reply:
> Sorry, a little bit confused, shouldn't it be "self._transfer_id" (with a preceding underscore) as in lines 57-58?

Both work, we have a @property exposing self._transfer_id as self.transfer_id. I think
most code uses the public properties.